minerl/herobraine/hero/handlers/agent/reward.py

```diff
@@ -93,14 +101,20 @@ def from_universal(self, obs):
 
 
 class RewardForCollectingItems(_RewardForPosessingItemBase):
-    def __init__(self, item_rewards: List[Dict[str, Union[str, int]]]):
-        """
+    """
         The standard malmo reward for collecting item.
 
-        rc = handlers.RewardForCollectingItems([
+        Example usage:
```

Comment:
Consistency thing: the docstring text should begin at same indent as the `"""` (i.e. remove one level of indent in this docstring).